app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,27 @@
   </head>
 
   <body>
+    <nav>
+      <div class="container">
+        <div class="row">
+          <div class="col-4 left">
+            <%= link_to "Pets", "/pets/"%> 
+          </div>
+          <div class="col-4 left">
+            <%= link_to "Shelters", "/shelters/"%> 
+          </div>
+          <div class="col-4 left">
+            <p>Favorited Pets: <%= favorite.total_count %></p>
```

Comment:
Very excited about this application view, I think this will save us time in the future. Excited to add more to this page.